src/Network/Network/VirtualNetworkGateway/StartAzVirtualNetworkGatewayPacketCaptureCommand.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network.VirtualNetworkGateway
+{
+    [Cmdlet("Start", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VirtualnetworkGatewayPacketCapture", 
+        DefaultParameterSetName = "ByName", SupportsShouldProcess = true), OutputType(typeof(PSVirtualNetworkGatewayPacketCaptureResult))]
```

Comment:
What is in the PacketCaptureReult?  Unless the user needs this data to obtain the captured packets, this likely should return nothing on success, and have a PassThru parameter to return a value on success.

Reply:
Packet Capture Result will contain command specific error codes and also in some cases result text may contain SASUrl where the packet capture is uploaded or error descriptions and fix suggestions. Packet Capture result will not contain packet information. As part of pass thru we expect the the consumer to make use of this result object if an diagnostic orchestration is being written over the command.